### PR TITLE
Change verbage to reflect Resonant branding

### PR DESCRIPTION
**Pull request by marySalvi**

### CODE REVIEW

File: doc/source/explanation/index.rst [hunk 1]
```diff
@@ -22,6 +22,6 @@ There are also support materials for
...
 
-And remember that Girder is still Girder! We are able to do the same rich application building and problem solving you may be used to with Girder 3, but we have learned many lessons and improved our process, knowledge, and understanding in moving to Girder 4. And of course, we are still supporting Girder 3 projects that already exist—we are simply building *new* projects with the enhanced insight and understanding we have captured in the Girder 4 family of tools.
+We are able to do the same rich application building and problem solving you may be used to with Resonant, but we have learned many lessons and improved our process, knowledge, and understanding in moving to Resonant. And of course, we are still supporting Girder  projects that already exist—we are simply building *new* projects with the enhanced insight and understanding we have captured in the Resonant family of tools.
```
Comment (marySalvi) [inline doc/source/explanation/index.rst:27]: > And of course, we are still supporting Girder  projects that already exist—we are simply building *new* projects with the enhanced insight and understanding we have captured in the Resonant family of tools.

I don't know if this is still accurate. Are we expecting new Girder 3 projects? If so should this sentence just be removed?

File: doc/source/explanation/index.rst [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+**Resonant** is the Kitware Data and Analytics team’s newest framework we use to create rich web applications and APIs tailored to specific kinds of data and tasks. Girder, another framework created by the Data and Analytics team, used the CherryPy web server framework to present a monolithic platform upon which to build specific applications. Resonant marks a technology shift that enables us to build those applications more effectively: it is based instead upon Django, a more expansive web application framework with out-of-the-box support for many of the nuts and bolts of web application engineering. Django’s integrated support for domain models backed by a comprehensive object-relational mapping (ORM) means that Resonant projects place *domain-driven design* front and center, enabling our team to better understand and address the computing and data problems we take on, so we can deliver the right solution every time.
```
Comment (marySalvi) [inline doc/source/explanation/index.rst:10]: Please look over this paragraph in particular.

File: doc/source/explanation/index.rst (hunk 1 above)
Comment (brianhelba) [reply doc/source/explanation/index.rst:27]: I think we can cut this entire line. It just muddies the water.

Comment (brianhelba) [inline doc/source/explanation/index.rst:27]:
```suggestion
```
I think we can cut this entire line. It just muddies the water.

File: doc/source/explanation/index.rst (hunk 2 above)
Comment (brianhelba) [inline doc/source/explanation/index.rst:10]:
```suggestion
**Resonant** is the Kitware Data and Analytics team’s newest framework, which we use to create rich web applications and APIs tailored to specific kinds of data and tasks. Girder, another framework created by the Data and Analytics team, used the CherryPy web server framework to present a monolithic platform upon which to build specific applications. Resonant marks a technology shift that enables us to build those applications more effectively: it is based instead upon Django, a more expansive web application framework with out-of-the-box support for many of the nuts and bolts of web application engineering. Django’s integrated support for domain models backed by a comprehensive object-relational mapping (ORM) means that Resonant projects place *domain-driven design* front and center, enabling our team to better understand and address the computing and data problems we take on, so we can deliver the right solution every time.
```

Comment (brianhelba) [reply doc/source/explanation/index.rst:10]: Looks fine, except for one grammar suggestion.

**Pull request merged**